packages/web/screens/Vitrine/Institution/index.js

```diff
@@ -98,13 +103,18 @@ InstitutionShowcasePage.propTypes = {
 		email: PropTypes.string.isRequired,
 		website: PropTypes.string.isRequired,
 	}).isRequired,
+	notFound: PropTypes.bool,
+};
+
+InstitutionShowcasePage.defaultProps = {
+	notFound: false,
 };
 
-InstitutionShowcasePage.getInitialProps = async ({ query, res, asPath }) => {
+InstitutionShowcasePage.getInitialProps = async ({ query, asPath }) => {
 	const institution = await getInstitution(query.institution);
 
 	if (!institution) {
-		return res.writeHead(302, { Location: '/_error.js' }).end();
+		return { notFound: true };
```

Comment:
This would crash in client-side cuz there's no res. As we can't use `getServerSideProps` due a bug in `react-instantsearch-dom`, this seems to be the most appropriate way to solve a not found institution.